Sources/Apodini/Information/Information.swift

```diff
@@ -7,36 +7,54 @@
 
 import Foundation
 
+/// Describes a type erased version of an `Information` instance.
+public protocol AnyInformation {
+    /// Internal method turning an `Information` instance into a `AnyInformationEntry`.
+    func entry() -> AnyInformationEntry
```

Comment:
AnyInformationEntry is a struct not a protocol.
Though it should ne possible to just remove this requirement in the public protocol definition and just declare the extension with he implementation internal, as this func is used internally only.

Further, when I rethink the AnyInformationEntry, I may be able to remove the need for this function anyways 🤔

Reply:
Yes, that was more of the direction I was thinking off. Either make `AnyInformationEntry` a protocol or remove it entirely by using a map for storage instead of a set.

Reply:
Removed with the latest refactoring.